plugins/connection/httpapi.py

```diff
@@ -140,13 +152,15 @@
 
 from ansible.errors import AnsibleConnectionFailure
 from ansible.module_utils._text import to_bytes
+from ansible.module_utils.compat.version import StrictVersion
```

Comment:
Unfortunately this is only available in ansible-core>=2.12. You can use `Version` from `ansible_collections.ansible.netcommon.plugins.plugin_utils.version` instead which doesn't do nearly as much but should be good enough here